docs/README.md

```diff
@@ -2957,6 +2958,76 @@ Variables
 
 The databases for the local provider are just json files in the `<project-root>/.booster` folder. If you are wondering what data is available in the application you will only need to chose what file to look into.
 
+### Testing Booster applications
+To properly test a Booster application, you should create a `test` folder at the same level as the `src` one (check the generated `tsconfig.json` for more information). Apart from that, tests' names should have the `<my_test>.test.ts` format.
+
+#### Testing with sinon-chai
+The `BoosterConfig` can be accessed through the `Booster.config` on any part of a Booster application. To properly mock it for your objective, we really recommend to use sinon `replace` method, after configuring your `Booster.config` as desired.
+
+In the example below, we add 2 "empty" read-models, since we are iterating `Booster.config.readModels` from a command handler:
+
+```typescript
+// Test
+import { replace } from 'sinon'
+
+const config = new BoosterConfig('test')
+config.appName = 'testing-time'
+config.provider = AWSProvider.Provider()
+config.readModels['WoW'] = {} as ReadModelMetadata
+config.readModels['Amazing'] = {} as ReadModelMetadata
+replace(Booster, 'config', config)
+
+const spyMyCall = spy(MyCommand, 'myCall')
+const command = new MyCommand('1', true)
+const register = new Register('request-id-1')
+const registerSpy = spy(register, 'events')
+await MyCommand.handle(command, register)
+
+expect(spyMyCall).to.have.been.calledOnceWithExactly('WoW')
+expect(spyMyCall).to.have.been.calledOnceWithExactly('Amazing')
+expect(registerSpy).to.have.been.calledOnceWithExactly(new MyEvent('1', 'WoW'))
+expect(registerSpy).to.have.been.calledOnceWithExactly(new MyEvent('1', 'Amazing'))
+```
+
+```typescript
+// Example code
+public static async handle(command: MyCommand, register: Register): Promise<void> {
+  const readModels = Booster.config.readModels
+  for (const readModelName in readModels) {
+    myCall(readModelName)
+    register.events(new MyEvent(command.ID, readModelName))
+  }
+}
+```
+
+#### Recommended files
+These are some files that might help you speed up your testing with Booster.
+
+```typescript
+// <root_dir>/test/expect.ts
```

Comment:
Are we providing this? Where the expect comes from?

Reply:
No we are not, this is a way of allowing the user to use more specific methods from `sinon-chai` and `chai-as-promised`. It's not mandatory but very useful for testing :v: @charlietfe